docs/cluster_management.md

```diff
@@ -2,45 +2,152 @@
 
 This doc is in progress.
 
+## Overview
+
+Kubernetes is an orchestration system for managing container clusters. This document provides basic instructions for setting up and managing the cluster and assumes familiarity with Kubernetes architecture and terminology.
```

Comment:
Overview not needed.  This doc should be linked-to by some other higher-level doc which gives enough context.